win32/tests/interlocked_win32_ut/interlocked_win32_ut.c

```diff
@@ -0,0 +1,741 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifdef __cplusplus
+#include <cstdlib>
+#include <cstddef>
+#else
+#include <stdlib.h>
+#include <stddef.h>
+#include <stdbool.h>
+#endif
+#include "windows.h"
+#include "azure_macro_utils/macro_utils.h"
+#include "testrunnerswitcher.h"
+
+#include "umock_c/umock_c.h"
+#include "umock_c/umocktypes_windows.h"
+
+#include "interlocked.h"
```

Comment:
Leave as windows.h for now

Reply:
If I remove this include, then the snake_case interlocked functions won't be found. Since the tests are already written for the snake_case functions, I think it will be easier to leave it as it is.

---
In reply to: [437079405](https://github.com/Azure/azure-c-pal/pull/1#discussion_r437079405) [](ancestors = 437079405)